src/net_processing.cpp

```diff
@@ -3856,9 +3856,10 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
                         const uint256& hash = txinfo.tx->GetHash();
                         CInv inv(MSG_TX, hash);
                         pto->m_tx_relay->setInventoryTxToSend.erase(hash);
-                        if (filterrate) {
-                            if (txinfo.feeRate.GetFeePerK() < filterrate)
+                        if (filterrate > CFeeRate(0)) {
```

Comment:
μ-Nit: I would slightly prefer a `IsNull() { return nSatoshisPerK == 0; }` in `CFeeRate` and using that instead of instantiating `CFeeRate()`s whenever you check if the filter rate is set.

Reply:
will do if forced to rebase or change commits for more substantial reasons

Reply:
maybe isZero instead of isNull? I always associate Null with pointers somehow.

Reply:
Either works for me. We use `IsNull` elsewhere (e.g. `uint256`, `FlatFilePos`, `BlockHeader`, `PSBTInput`, `CScriptWitness`, etc etc.)

Reply:
Same goes for `uint256` I'd say, but I don't really have a strong opinion on the matter. (Consistency -> IsNull; intuitive -> IsZero.)